docs/30-simple-queries/2-match.mdx

```diff
@@ -26,7 +26,7 @@ The `<expression>` portion of the $match operator can be any valid MongoDB expre
     First, make sure you select the `books` collection in the Atlas UI. 
     <Screenshot src="/img/30-simple-queries/select-books-collection.png" url="http://cloud.mongodb.com/" alt="Atlas UI database deployment with the books collection highlighted." />
 
-    Then, navigate to the `Aggregation` tab and click `Add Stage`.
+    Then, navigate to the `Aggregation` tab and click `</> TEXT`.
```

Comment:
The original instruction doesn't match the screenshot. I think this is what was intended